[2022-08-06 19:39:06,254] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-30T00:00:00+00:00 [queued]>
[2022-08-06 19:39:06,270] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-30T00:00:00+00:00 [queued]>
[2022-08-06 19:39:06,270] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:39:06,270] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 19:39:06,270] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:39:06,288] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-05-30 00:00:00+00:00
[2022-08-06 19:39:06,295] {standard_task_runner.py:52} INFO - Started process 2982 to run task
[2022-08-06 19:39:06,298] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-05-30T00:00:00+00:00', '--job-id', '919', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmphoitb6b7', '--error-file', '/tmp/tmp_q5g05m3']
[2022-08-06 19:39:06,298] {standard_task_runner.py:80} INFO - Job 919: Subtask second_task
[2022-08-06 19:39:06,370] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-30T00:00:00+00:00 [running]> on host b29c168c1666
[2022-08-06 19:39:06,475] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-05-30T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-30T00:00:00+00:00
[2022-08-06 19:39:06,476] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 19:39:06,477] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 19:39:06,492] {subprocess.py:85} INFO - Output:
[2022-08-06 19:39:06,494] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 19:39:06,495] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 19:39:06,540] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220530T000000, start_date=20220806T193906, end_date=20220806T193906
[2022-08-06 19:39:06,589] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 19:39:06,642] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:52:57,357] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-30T00:00:00+00:00 [queued]>
[2022-08-06 21:52:57,372] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-30T00:00:00+00:00 [queued]>
[2022-08-06 21:52:57,372] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:52:57,372] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:52:57,372] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:52:57,390] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-05-30 00:00:00+00:00
[2022-08-06 21:52:57,398] {standard_task_runner.py:52} INFO - Started process 3160 to run task
[2022-08-06 21:52:57,401] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-05-30T00:00:00+00:00', '--job-id', '918', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpzqmvg5q_', '--error-file', '/tmp/tmpn_hbn0t7']
[2022-08-06 21:52:57,401] {standard_task_runner.py:80} INFO - Job 918: Subtask second_task
[2022-08-06 21:52:57,502] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-30T00:00:00+00:00 [running]> on host c3f731879890
[2022-08-06 21:52:57,583] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-05-30T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-30T00:00:00+00:00
[2022-08-06 21:52:57,583] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:52:57,584] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:52:57,597] {subprocess.py:85} INFO - Output:
[2022-08-06 21:52:57,598] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:52:57,598] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:52:57,629] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220530T000000, start_date=20220806T215257, end_date=20220806T215257
[2022-08-06 21:52:57,654] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:52:57,684] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:58:25,145] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-30T00:00:00+00:00 [queued]>
[2022-08-06 21:58:25,159] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-30T00:00:00+00:00 [queued]>
[2022-08-06 21:58:25,159] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:58:25,159] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:58:25,159] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:58:25,176] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-05-30 00:00:00+00:00
[2022-08-06 21:58:25,184] {standard_task_runner.py:52} INFO - Started process 2971 to run task
[2022-08-06 21:58:25,187] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-05-30T00:00:00+00:00', '--job-id', '920', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp7e2ae_4s', '--error-file', '/tmp/tmpwf4gxqme']
[2022-08-06 21:58:25,188] {standard_task_runner.py:80} INFO - Job 920: Subtask second_task
[2022-08-06 21:58:25,282] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-30T00:00:00+00:00 [running]> on host bc7e154a4fdd
[2022-08-06 21:58:25,392] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-05-30T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-30T00:00:00+00:00
[2022-08-06 21:58:25,394] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:58:25,394] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:58:25,413] {subprocess.py:85} INFO - Output:
[2022-08-06 21:58:25,417] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:58:25,418] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:58:25,459] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220530T000000, start_date=20220806T215825, end_date=20220806T215825
[2022-08-06 21:58:25,520] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:58:25,559] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:13:23,410] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-30T00:00:00+00:00 [queued]>
[2022-08-06 22:13:23,422] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-30T00:00:00+00:00 [queued]>
[2022-08-06 22:13:23,422] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:13:23,422] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:13:23,422] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:13:23,438] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-05-30 00:00:00+00:00
[2022-08-06 22:13:23,445] {standard_task_runner.py:52} INFO - Started process 2973 to run task
[2022-08-06 22:13:23,447] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-05-30T00:00:00+00:00', '--job-id', '920', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmprrquttbk', '--error-file', '/tmp/tmpj6qltor_']
[2022-08-06 22:13:23,448] {standard_task_runner.py:80} INFO - Job 920: Subtask second_task
[2022-08-06 22:13:23,517] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-30T00:00:00+00:00 [running]> on host 1250c3b659cd
[2022-08-06 22:13:23,612] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-05-30T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-30T00:00:00+00:00
[2022-08-06 22:13:23,613] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:13:23,613] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:13:23,628] {subprocess.py:85} INFO - Output:
[2022-08-06 22:13:23,630] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:13:23,630] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:13:23,666] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220530T000000, start_date=20220806T221323, end_date=20220806T221323
[2022-08-06 22:13:23,699] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:13:23,750] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:24:43,313] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-30T00:00:00+00:00 [queued]>
[2022-08-06 22:24:43,324] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-30T00:00:00+00:00 [queued]>
[2022-08-06 22:24:43,324] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:24:43,324] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:24:43,325] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:24:43,341] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-05-30 00:00:00+00:00
[2022-08-06 22:24:43,348] {standard_task_runner.py:52} INFO - Started process 3030 to run task
[2022-08-06 22:24:43,351] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-05-30T00:00:00+00:00', '--job-id', '922', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp0gyupaul', '--error-file', '/tmp/tmp_0o7mnki']
[2022-08-06 22:24:43,351] {standard_task_runner.py:80} INFO - Job 922: Subtask second_task
[2022-08-06 22:24:43,428] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-30T00:00:00+00:00 [running]> on host b215076695c3
[2022-08-06 22:24:43,553] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-05-30T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-30T00:00:00+00:00
[2022-08-06 22:24:43,554] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:24:43,555] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:24:43,569] {subprocess.py:85} INFO - Output:
[2022-08-06 22:24:43,571] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:24:43,571] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:24:43,663] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220530T000000, start_date=20220806T222443, end_date=20220806T222443
[2022-08-06 22:24:43,723] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:24:43,771] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
